internal/controllers/core/clustercatalog_controller.go

```diff
@@ -168,7 +163,8 @@ func (r *ClusterCatalogReconciler) reconcile(ctx context.Context, catalog *v1alp
 			lastUnpacked = metav1.Time{}
 		}
 
-		updateStatusUnpacked(&catalog.Status, unpackResult, contentURL, catalog.Generation, lastUnpacked)
+		_ = updateStatusProgressing(catalog, nil)
```

Comment:
Are we intentionally not checking error here? If yes, then we can just do that, right?
```suggestion
		updateStatusProgressing(catalog, nil)
```

Reply:
updateStatusProgressing was returning the error that was passed in unchanged, modified the function to avoid this